### DISCUSSION TITLE
The block ID generated by this plug-in is after the paragraph, not after the block. Therefore, the link cannot be used to jump to the original text. This tip is "Unable to find section #^2znmr in XXX".

### DISCUSSION THREAD

**Discussion by liz666** [jglev/obsidian-carry-forward#35] [Q&A]
The block ID generated by this plug-in is after the paragraph, not after the block. Therefore, the link cannot be used to jump to the original text. This tip is "Unable to find section #^2znmr in XXX".

**Comment by jglev** [comment]
Thank you for reaching out!

Would you be willing to provide an example block where this behavior occurs?
